src/AppLayout.vue

```diff
@@ -30,6 +30,7 @@
 
 <script>
 import Vue from 'vue';
+import camelCase from 'lodash.camelcase';
```

Comment:
It'll be better to use `lodash/camelCase` instead, that way it won't be duplicated (we're using that format everywhere).

Reply:
Done